Source/WebKit/Platform/IPC/unix/UnixMessage.h

```diff
@@ -82,7 +82,6 @@ class UnixMessage {
             memcpy(m_body, other.m_body, m_messageInfo.bodySize());
             m_bodyOwned = true;
             other.m_body = nullptr;
-            other.m_bodyOwned = false;
```

Comment:
I don't see what's wrong about trying to clean up m_bodyOwned in the object used as source of the move constructor.

Reply:
Nothing wrong about it, it's not needed though:
        if (other.m_bodyOwned) {
            //...
        } else /*if (... doesn't matter)*/ {
            //...
            other.m_bodyOwned = false; //it already *is* false
        }

I can drop this part if you prefer

Reply:
Oh! Now I see after expanding the diff context. Sorry, you're right, please disregard my comment.